config/database.yml

```diff
@@ -1,6 +1,8 @@
 default: &default
   adapter: postgresql
-  host: localhost
+  host: <%= ENV['GLOWFIC_DATABASE_HOST'] || 'localhost' %>
+  username: <%= ENV['GLOWFIC_DATABASE_USER'] || 'root' %>
```

Comment:
I think dev servers and Travis disagree unfortunately: https://docs.travis-ci.com/user/database-setup/#postgresql might just need to use the config files that suggests

Reply:
I was actually able to convince it to use environment variables so that travis would see one config and docker would see another one. Also shouldn't break anyone's existing dev environment.